### ISSUE TITLE
Spring boot M7 compatibility

### ISSUE THREAD

**Issue by wwadge** [gclayburg/dockerPreparePlugin#4]
Hi,

Can you update to fix for Spring boot M7 compatibility? I get : 

```
Caused by: java.lang.NullPointerException: Cannot invoke method isIncluded() on null object
        at com.garyclayburg.docker.DockerPreparePlugin.failWhenExecutable(DockerPreparePlugin.groovy:156)
        at com.garyclayburg.docker.DockerPreparePlugin$_apply_closure1.doCall(DockerPreparePlugin.groovy:56)

```

**Comment by gclayburg** [comment]
I'll take a look.  Could you show me the build.gradle file you are using?   I'd like to create a test case from it.

**Comment by gclayburg** [comment]
I just created a fix for this and published it as a new version, 1.1.1:

plugins {
  id "com.garyclayburg.dockerprepare" version "1.1.1"
}

Does it work for you, @wwadge ?

**Comment by gclayburg** [comment]
FYI - this is fixed now.
